brax/physics/colliders.py

```diff
@@ -27,6 +27,27 @@
 from jax import ops
 import jax.numpy as jnp
 
+@struct.dataclass
```

Comment:
I'm not sure this dataclass is adding much as a grouping concept.  I can't find anywhere in the code where you are referencing `Collider.idx`.  How about just treating friction as a separate scalar for now?

In some future PR we'll clean up these groupings a bit.  They are a bit arbitrarily different at the moment (e.g. CapsuleCapsule has its own struct, while BoxPlane just has a bunch of fields, etc)

Reply:
Yes, separate friction scalar for now would make sense. I was trying to minimize the number of future changes to the signatures of `_collide()` etc., but I think keeping things simple for now would be best.